### PR TITLE
EZP-31479: Added Tooltips to UIGuidelines v3.0

### PR DESCRIPTION
**Pull request by inakijv**
| Question      | Answer
| ------------- | ---
| JIRA Ticket   | [EZP-31479](https://jira.ez.no/browse/EZP-31479)
| Versions      | 3.0

Tooltips - eZ Platform Developer Documentation

### CODE REVIEW

**Comment by inakijv** [comment]
Ping @ezsystems/documentation-team @dew326 @lucasOsti 
Thank you @damianz5 for the internal review 👍

File: docs/guidelines/components/tooltips.md [hunk 1]
```diff
@@ -0,0 +1,133 @@
+# Tooltips
+
+We use Tooltips to provide more information for the users to identify an element, such as a description of its function, and guide them to take action.
+
+**<div class="mgt-3">How they work</div>**
+To add a new tooltip you have to add a `title` attribute with the custom tooltip text in the HTMLnode you need to have a tooltip.
```
Comment (DominikaK) [inline docs/guidelines/components/tooltips.md]:
```suggestion
To add a new tooltip you have to add a `title` attribute with the custom tooltip text in the HTMLnode that should have a tooltip.
```

File: docs/guidelines/components/tooltips.md [hunk 2]
```diff
@@ -0,0 +1,133 @@
...
+
+```html
+<button type="button" class="class" title="your custom tooltip text">click me</button>
+```
+
+Tooltip is displayed automatically when users hover the pointer over an action button, and removed when users click the control or move the mouse. Tooltip can also be triggered by focusing on that specific element with the keyboard (tab key), or tap on it.
```
Comment (DominikaK) [inline docs/guidelines/components/tooltips.md]:
```suggestion
A tooltip is displayed automatically when the user hovers the pointer over an action button,
and removed when the user clicks the control or moves the mouse.
The tooltip can also be triggered by focusing on that specific element with the keyboard (tab key), or tapping on it.
```

File: docs/guidelines/components/tooltips.md [hunk 3]
```diff
@@ -0,0 +1,133 @@
...
+
+!!! Note
```
Comment (DominikaK) [inline docs/guidelines/components/tooltips.md]:
```suggestion
!!! note

```

File: docs/guidelines/components/tooltips.md [hunk 4]
```diff
@@ -0,0 +1,133 @@
...
+    Our application relies on [Bootstrap's tooltips](https://getbootstrap.com/docs/4.1/components/tooltips/). Check our their documentation for basic aspects regarding Tooltips configuration.</br>
+    <div class="mgt-2">If you are interested in customizing or extending this component, check out [Extending Back Office, Add tooltips](guide/extending/extending_back_office.md#add-tooltips).</div>
```
Comment (DominikaK) [inline docs/guidelines/components/tooltips.md]:
```suggestion
    <div class="mgt-2">If you are interested in customizing or extending this component, check out [Extending Back Office](../../guide/extending/extending_back_office.md#add-tooltips).</div>
```

File: docs/guidelines/components/tooltips.md [hunk 5]
```diff
@@ -0,0 +1,133 @@
...
+
+**<div class="mgt-3">Behavior & motion</div>**
+A tooltip is displayed upon hovering over an action button. They show up over 150ms and fade out over 75ms.
```
Comment (DominikaK) [inline docs/guidelines/components/tooltips.md]:
```suggestion
A tooltip is displayed upon hovering over an action button. It shows up over 150ms and fades out over 75ms.
```

**Comment by inakijv** [comment]
Updated based on feedback.

**Pull request merged**